lib/cocoapods-binary/Main.rb

```diff
@@ -64,6 +64,12 @@ def keep_source_code_for_prebuilt_frameworks!
     
     Pod::UI.puts "🚀  Prebuild frameworks"
     
+    # Looks like this is the most appropriate way to figure out that something should be updated
+    include ObjectSpace
+    ObjectSpace.each_object(Pod::Installer) { |installer|
+        @update = installer.update
```

Comment:
I think instance variable is unnecessary here.  There's no meaning of saving info here. A local variable is just recommended.